bin/sdc-thoth-install

```diff
@@ -92,10 +95,10 @@ if [[ -z $DOWNLOAD_MANTA ]]; then
 
 fi
 
-[[ -n $SDC_ACCOUNT ]] || fatal "must set \$SDC_ACCOUNT. This is the account in manta that stores your thoth data."
-[[ -n $SDC_KEY_ID ]] || fatal "must set \$SDC_KEY_ID. This must already exist in the $SDC_ACCOUNT account, and needs to be available on this node (ssh-agent is recommended)."
+[[ -n $TRITON_ACCOUNT ]] || fatal "must set \$TRITON_ACCOUNT. This is the account in manta that stores your thoth data."
```

Comment:
Since TRITON_* is the documented interface in the docs, I'd rather use those.

Reply:
Yeah, I'm not even sure what the best thing to do here is, because we're just at the mercy of node-triton's behavior. I just wanted to note it here at least.

Reply:
> and SDC_* vars are still preferred if present.

Really? Nope, `TRITON_*` wins over a `SDC_*` var setting.
https://github.com/joyent/node-triton/blob/6775738bb422dff54dd843c560fffc5eb7730f71/lib/config.js#L279-L297

Reply:
Oh, good. I thought it was the other way around for some reason.